src/evm/tycho_models.rs

```diff
@@ -1,115 +1,14 @@
 
-use super::engine_db::simulation_db::BlockHeader;
 use crate::{
     evm::protocol::u256_num,
     serde_helpers::{hex_bytes, hex_bytes_option},
 };
 
-#[derive(Serialize, Deserialize, Debug, Clone, PartialEq, Eq, Hash, Default)]
-pub struct ExtractorIdentity {
-    pub chain: Chain,
-    pub name: String,
-}
-
-impl ExtractorIdentity {
-    pub fn new(chain: Chain, name: &str) -> Self {
-        Self { chain, name: name.to_owned() }
-    }
-}
-
-impl std::fmt::Display for ExtractorIdentity {
-    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        write!(f, "{}:{}", self.chain, self.name)
-    }
-}
-
-/// A command sent from the client to the server
-#[derive(Deserialize, Serialize, Debug, PartialEq, Eq)]
-#[serde(tag = "method", rename_all = "lowercase")]
-pub enum Command {
-    Subscribe { extractor_id: ExtractorIdentity },
-    Unsubscribe { subscription_id: Uuid },
-}
-
-/// A response sent from the server to the client
-#[derive(Deserialize, Serialize, Debug, PartialEq, Eq)]
-#[serde(tag = "method", rename_all = "lowercase")]
-pub enum Response {
-    NewSubscription { extractor_id: ExtractorIdentity, subscription_id: Uuid },
-    SubscriptionEnded { subscription_id: Uuid },
-}
-
-/// A message sent from the server to the client
-#[derive(Serialize, Deserialize, Debug)]
-#[serde(untagged)]
-pub enum WebSocketMessage {
-    BlockAccountChanges(BlockAccountChanges),
-    Response(Response),
-}
-
-#[derive(Debug, PartialEq, Copy, Clone, Deserialize, Serialize, Default)]
-pub struct Block {
-    pub number: u64,
-    pub hash: B256,
-    pub parent_hash: B256,
-    pub chain: Chain,
-    pub ts: NaiveDateTime,
-}
-
-impl From<Block> for BlockHeader {
-    fn from(value: Block) -> Self {
-        Self {
-            number: value.number,
-            hash: value.hash,
-            timestamp: value.ts.and_utc().timestamp() as u64,
-        }
-    }
-}
-
-#[derive(Serialize, Deserialize, PartialEq, Debug, Clone)]
-pub struct SwapPool {}
-
-#[derive(Debug, PartialEq, Copy, Clone, Default, Deserialize, Serialize)]
-pub struct Transaction {
-    pub hash: B256,
-    pub block_hash: B256,
-    pub from: Address,
-    pub to: Option<Address>,
-    pub index: u64,
-}
-
-/// A container for account updates grouped by account.
-///
-/// Hold a single update per account. This is a condensed form of
-/// [BlockStateChanges].
-#[derive(Debug, Clone, PartialEq, Deserialize, Serialize, Default)]
-pub struct BlockAccountChanges {
-    extractor: String,
-    chain: Chain,
-    pub block: Block,
-    pub account_updates: HashMap<Address, AccountUpdate>,
-    pub new_pools: HashMap<Address, SwapPool>,
-}
-
-impl BlockAccountChanges {
-    pub fn new(
-        extractor: String,
-        chain: Chain,
-        block: Block,
-        account_updates: HashMap<Address, AccountUpdate>,
-        new_pools: HashMap<Address, SwapPool>,
-    ) -> Self {
-        Self { extractor, chain, block, account_updates, new_pools }
-    }
-}
-
```

Comment:
now in this file we only have AccountUpdate  and ResponseAccount. I actually think they should be in tycho-common even though they are used only here 🤔 they are just a wrapper around the dto objects. what do you think?